routes/web.php

```diff
@@ -43,5 +43,9 @@
 //釣り記録詳細ページ
 Route::get('fish_records/{id}','FishRecordController@detalis')->name('detalis');
 
+//釣り記録詳細ページのコメント
+Route::post('fish_records/{id}','FishRecordController@comment')->name('comment');
```

Comment:
釣り履歴の登録や更新用のルーティングに見えるので下記のほうが良いかなと思いました
```suggestion
Route::post('fish_records/{id}/comments','FishRecordController@comment')->name('comment');
```